requirements.txt

```diff
@@ -0,0 +1,8 @@
+numpy
+scipy
+pyqt5
+pillow
+h5py
+matplotlib
+pyvtk
+vtk
```

Comment:
@RemiLehe isn't `openpmd-viewer` missing in the list for `openPMD` file handling?

Reply:
It is not a strict requirement: some users of visualPIC are using it for Osiris or HiPACE, in which case `openPMD-viewer` is not required.

Reply:
yes, but you could add it as a `[openpmd]` subpackage.

also, since the `openpmd-viewer` core dependencies for the api are only `numpy`, `scipy`, `h5py` which are also dependencies of VisualPIC it should be fine to add it by default, no?

Reply:
OK, that's true. I'll add it ; but ultimately I guess it's @AngelFP 's call.

Reply:
Absolutely, @AngelFP what do you think?